From: Josef Bacik <josef@redhat.com>
Date: Thu, 14 Jul 2011 14:28:08 -0400
Patch-mainline: pending
References: FATE#306586
Git-commit-unstable: f6c0072f1ebaaa8ff0b2b2140925e8f5906783d0
Subject: [PATCH] Btrfs: move stuff around in btrfs_inode to get better
 packing

Moving things around to give us better packing in the btrfs_inode.  This reduces
the size of our inode by 8 bytes.  Thanks,

Signed-off-by: Josef Bacik <josef@redhat.com>
Signed-off-by: David Sterba <dsterba@suse.cz>
---
 fs/btrfs/btrfs_inode.h |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/fs/btrfs/btrfs_inode.h b/fs/btrfs/btrfs_inode.h
index 502b9e9..e18dfcb 100644
--- a/fs/btrfs/btrfs_inode.h
+++ b/fs/btrfs/btrfs_inode.h
@@ -115,9 +115,6 @@ struct btrfs_inode {
 	 */
 	u64 disk_i_size;
 
-	/* flags field from the on disk inode */
-	u32 flags;
-
 	/*
 	 * if this is a directory then index_cnt is the counter for the index
 	 * number for new files that are created
@@ -131,6 +128,9 @@ struct btrfs_inode {
 	 */
 	u64 last_unlink_trans;
 
+	/* flags field from the on disk inode */
+	u32 flags;
+
 	/*
 	 * Counters to keep track of the number of extent item's we may use due
 	 * to delalloc and such.  outstanding_extents is the number of extent
-- 
1.7.6

